Sources/BraintreeCard/BTCardClient.swift

```diff
@@ -0,0 +1,218 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Used to process cards
+@objc public class BTCardClient: NSObject {
+
+    // MARK: - Internal Properties
+
+    /// Exposed for testing to get the instance of BTAPIClient
+    let apiClient: BTAPIClient
+
+    let graphQLTokenizeFeature: String = "tokenize_credit_cards"
+
+    // MARK: - Initializer
+
+    /// Creates a card client
+    /// - Parameter apiClient: An API client
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    // MARK: - Public Methods
+
+    /// Tokenizes a card
+    /// - Parameters:
+    ///    - card: The card to tokenize.
+    ///    - completion: A completion block that is invoked when card tokenization has completed. If tokenization succeeds,
+    ///    `tokenize` will contain a nonce and `error` will be `nil`; if it fails, `tokenize` will be `nil` and `error`will describe the failure.
+    @objc(tokenizeCard:completion:)
+    public func tokenize(_ card: BTCard, completion: @escaping (BTCardNonce?, Error?) -> Void) {
+        let request = BTCardRequest(card: card)
+
+        apiClient.fetchOrReturnRemoteConfiguration() { configuration, error in
+            guard let configuration, error == nil else {
+                completion(nil, error)
```

Comment:
Are we going to send BTCardAnalytics.cardTokenizedFailed?

Reply:
Same answer as here - the updated analytics names don't exist on `master` yet: https://github.com/braintree/braintree_ios/pull/983#discussion_r1145098549